frontend/talentsearch/src/js/components/browse/PoolCard.tsx

```diff
@@ -0,0 +1,178 @@
+          {pool.stream ? intl.formatMessage(getPoolStream(pool.stream)) : ""}
+        </h6>
+
+        <p data-h2-padding="base(0, 0, x0.75, 0)">
+          {pool.description ? pool.description[locale] : ""}
+        </p>
+        <p data-h2-padding="base(0, 0, x0.75, 0)">
+          <span
+            data-h2-font-weight="base(700)"
+            data-h2-padding="base(0, x0.25, 0, 0)"
+          >
+            {intl.formatMessage({
+              defaultMessage: "Salary range",
+              id: "MKyZVJ",
+            })}
+            :{" "}
+          </span>{" "}
+          {poolClassificationSalary()
+            ? poolClassificationSalary()
+            : intl.formatMessage({
+                defaultMessage: "(Needs classification)",
+                description: "Needs classification",
+                id: "nTGIfl",
+              })}
+        </p>
+
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(row)"
+          data-h2-align-items="base(center)"
+          data-h2-padding="base(0, 0, x0.75, 0)"
+        >
+          <span
+            data-h2-font-weight="base(700)"
+            data-h2-padding="base(0, x0.25, 0, 0)"
+          >
+            {intl.formatMessage({
+              defaultMessage: "Key skills",
+              description: "key skills",
+              id: "ppuSbM",
+            })}
+            :{" "}
+          </span>{" "}
+          <span>
+            <Chips>
+              {poolEssentialSkillKeys()
+                ? poolEssentialSkillKeys()
+                : intl.formatMessage({
+                    defaultMessage: "(No skills required)",
+                    description: "No skills required",
+                    id: "WbET08",
+                  })}
+            </Chips>
+          </span>
+        </div>
+
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(row)"
+          data-h2-justify-content="base(space-between)"
+          data-h2-align-items="base(center)"
+        >
+          <span>
+            {intl.formatMessage({
+              defaultMessage: "Apply by",
+              description: "Apply by",
+              id: "gh4VQv",
+            })}
+            :{" "}
+            {pool.expiryDate
+              ? formattedDateMonthDayYear(new Date(pool.expiryDate), intl)
+              : intl.formatMessage({
+                  defaultMessage: "(To be determined)",
+                  description: "To be determined",
+                  id: "3ozZky",
+                })}
+          </span>
+          <Link
+            href={pool.id ? paths.pool(pool.id) : "#"}
+            type="button"
+            mode="outline"
+            color="black"
+          >
+            {intl.formatMessage({
+              defaultMessage: "Apply to this recruitment",
+              description:
+                "Message on link that say to apply to a recruitment advertisement",
+              id: "zvn+Gu",
+            })}
```

Comment:
Just so we have a bit more context for this link, can we add a screen reader only part to the end of this string?

```tsx
{intl.formatMessage({
  defaultMessage: "Apply to recruitment<hidden> {name}</hidden>",
  description:
    "Message on link that say to apply to a recruitment advertisement",
  id: "zvn+Gu",
}, {
  name: cardTitle
})}
```

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/4233/commits/eb329dd298f5247f94f396666c3a054c773f889e